### DISCUSSION TITLE
server rollbacks possible?

### DISCUSSION THREAD

**Discussion by froghilda** [mbround18/valheim-docker#321] [Instructions]
Thank you for your prompt answer and the resource in how we can mod in lieu using this docker image to run our valheim server! <3

I just wonder one thing. Is there a how-to or guide on how to rollback a server. Let's say someone dc:ed, but respawned naked at reconnection, with no tombstone or any of his/her gear still existing, and we want to roll back to the last world save, let's say a few hours earlier. How would I go about making it so? I have auto back up set to true etc. in my docker-compose.yml

Thank you for any tip or guidance in fulfilling this use case (roll back server to previous world save point(s))!!

Froggy

**Comment by mbround18** [comment]
Unfortunately, character data is stored on the individuals computer and the server has no control over it. As for the tombstones, you could roll back a backup but that would undo any work you did before then.

**Reply by froghilda** [reply]
Ah, I got it, so all character data is stored client side. Thank you for that information! Then I would have to find a save where my latest tombstone was untouched and fresh, that sounds hard to try and pinpoint. Btw, have you ever experienced/heard of DC -> spawn naked at bed, and no trace of the gear anywhere in the world?

Thanks for your attention! Great image you got going here! Kudos!

Froggy

**Comment by JohnKreese** [comment]
If it's only one person then it would be easier to just respawn the stuff the person had than to rollback the whole server. The person can do this by starting new local game and enabling devcommands.

**Reply by froghilda** [reply]
Ahh smart! ..somehow it would feel a little bit like cheating (anyhow for me, personally).. I don't, and don't plan to "world-hop" for reasons related to immersion :) but thx for the tip! I will tell my co-players. :)

Froggy
